sway-lsp/src/traverse/parsed_tree.rs

```diff
@@ -624,8 +624,9 @@ impl<'a> ParsedTree<'a> {
                     self.collect_type_arg(type_arg, &token);
                 }
 
-                for exp in args {
-                    self.handle_expression(exp);
+                let args_vec = args.clone().unwrap_or_default();
+                for exp in args_vec {
```

Comment:
To eliminate the `.clone()`, change this block of code to something like:

```rust
args.map(|args_vec| {
  args_vec.iter().for_each(|exp| {
    self.handle_expression(&exp);
  })
});
```

Reply:
Followed the suggestion but then clippy told me to change it a bit.